mahalia/Jenkinsfile

```diff
@@ -1,39 +1,79 @@
+// Declarative Jenkins pipeline to build a realtime kernel for mahalia
 pipeline {
+
+  // This build needs a Jenkins agent that can cross-compile a linux kernel.
+  // Target of cross-compilation is armv7.  Refine labels when needed.
   agent {label "kernel && cross"}
+
   stages {
-    stage("build") {
+    stage("compile and package") {
       steps {
+        // The following build steps automate Chris' instructions from
+        // https://github.com/mahalia-dependencies/mahalia-utils/blob/master/compile_kernel/instructions.md
+
+        // Remove any leftovers from previous builds
         sh "make distclean"
+
+        // Build default configuration for beaglebone kernel with realtime patch
         sh "ti_config_fragments/defconfig_builder.sh -t ti_sdk_am3x_rt_release"
         sh "ARCH=arm make ti_sdk_am3x_rt_release_defconfig"
-        // Instead of  ARCH=arm make menuconfig we can copy an existing .config:
+
+        // We cannot manually execute make menuconfig in Jenkins. Instead, we
+        // store the resulting configuration file here in git and copy it to
```

Comment:
"store" sounds like a request to the reader of the comment. Instead, describe that the file is already stored in git.